### DISCUSSION TITLE
Template for pull-request

### DISCUSSION THREAD

**Discussion by JulienLefevreMars** [brain-slam/slam#77] [Ideas]
What are the main rules to contribute to slam through PR.

**Comment by alexpron** [comment]
Here is an  example on how to structure PR from nilearn (https://nilearn.github.io/stable/development.html#pr-structure)

**Reply by JulienLefevreMars** [reply]
It could be good to add a template for a .py file that is expected. I found no clear definitions of what is expected in a python functions, in terms of parameters and return, even here https://numpydoc.readthedocs.io/en/latest/format.html
Well, I am a bit lost :-)
